### PR TITLE
Disable unsupported featues on desktop

### PR DESCRIPTION
**Pull request by ShGKme**
### 🚧 TODO

- [x] Disable screensharing
- [x] Disable contacts menu on NcAvatar
  - [x] Check with new conversation avatar feature

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: src/components/AvatarWrapper/AvatarWrapper.vue [hunk 1]
```diff
@@ -129,6 +129,11 @@ export default {
 		sizeToString() {
 			return this.size.toString()
 		},
+		isDisabledMenu() {
+			// NcAvatarMenu doesn't work on Desktop
+			// See: issue#34
```
Comment (nickvergessen) [inline src/components/AvatarWrapper/AvatarWrapper.vue:134]:
```suggestion
			// See: https://github.com/nextcloud/talk-desktop/issues/34
```

File: src/components/AvatarWrapper/AvatarWrapperSmall.vue [hunk 2]
```diff
@@ -129,6 +129,11 @@ export default {
 		sizeToString() {
 			return this.size.toString()
 		},
+		isDisabledMenu() {
+			// NcAvatarMenu doesn't work on Desktop
+			// See: issue#34
```
Comment (nickvergessen) [inline src/components/AvatarWrapper/AvatarWrapperSmall.vue:134]:
```suggestion
			// See: https://github.com/nextcloud/talk-desktop/issues/34
```

Comment (nickvergessen) [reply src/components/AvatarWrapper/AvatarWrapperSmall.vue:134]: Same for the others

**Comment by ShGKme** [comment]
Checked with conversation avatar PR. It never uses the menu.

image

**Comment by ShGKme** [comment]
> Let the capability checking begin I guess?
> Similar to the mobile apps the frontend of talk should verify that the capability is there before presenting the feature.

No, works fine, conversation avatars don't affect using avatar menu.

**Review by Antreesy — APPROVED** [review]

**Pull request merged**